modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/view/table/PendingCommerceOrderTableFDSView.java

```diff
@@ -75,6 +76,22 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 		return stringFDSTableSchemaField;
 	}
 
+	private DateTimeFDSTableSchemaField _addDateFDSTableSchemaField() {
+		DateTimeFDSTableSchemaField dateFDSTableSchemaField =
+			new DateTimeFDSTableSchemaField();
+
+		dateFDSTableSchemaField.setFieldName(
+			"date"
+		).setLabel(
+			"create-date"
+		);
+
+		dateFDSTableSchemaField.setFormat(
+			JSONUtil.put("convertToUserTimeZone", true));
```

Comment:
Do we even need a flag for renderer? We can just do something like
```java
dateFDSTableSchemaField.setFormat(
    JSONUtil.put("timeZone", user.getTimeZone()));
``` 
We may want a frontend API that will leverage `Liferay.ThemeDisplay.getTimeZone()` in JS , but I'm not sure if it's needed for this task.

Reply:
I thought about this, yes. If we do this, then fields become dynamic, meaning we create another discrepancy between Java anf DSM datasets. To support this, TableSection would become `PROXY` elements instead of editable ones.

That led me to consider an static flag and frontend conversion